### PR TITLE
Sequence from disk

### PR DESCRIPTION
**Pull request by Chris7**
Support storing sequences on disk instead of in db.

I removed the bio crate in favor of noodles as we are extensively using that elsewhere. I tried a new pattern for making it simpler to expand the Sequence model in the future. I know we discussed the builder/default pattern, and one thing I think may point to the builder being better is it will have fewer copies and less use of to_string.

### CODE REVIEW

File: src/models.rs [hunk 1]
```diff
@@ -416,9 +416,9 @@ impl BlockGroup {
 
             let (contains_start, contains_end, overlap) =
                 get_overlap(path_start, path_end, start, end);
-            println!(
-                "{path_start} {path_end} {start} {end} {contains_start} {contains_end} {overlap}"
-            );
+            // println!(
```
Comment (dkhofer) [inline src/models.rs:419]: What's the reason for commenting this out?

File: src/models/block.rs [hunk 2]
```diff
@@ -251,18 +253,79 @@ impl Block {
     }
 
     pub fn get_sequence(conn: &Connection, block_id: i32) -> (String, String) {
-        let mut stmt = conn.prepare_cached("select substr(sequence.sequence, block.start + 1, block.end - block.start) as sequence, block.strand from sequence left join block on (block.sequence_hash = sequence.hash) where block.id = ?1").unwrap();
+        let mut stmt = conn.prepare_cached("select name, substr(sequence.sequence, block.start + 1, block.end - block.start) as sequence, block.start, block.end, block.strand, file_path from sequence left join block on (block.sequence_hash = sequence.hash) where block.id = ?1").unwrap();
         let mut rows = stmt
-            .query_map((block_id,), |row| Ok((row.get(0)?, row.get(1)?)))
+            .query_map((block_id,), |row| {
```
[the remark this answers is not in the document]
Comment (dkhofer) [reply src/models/block.rs:258]: Maybe edge_pairs_to_block: https://github.com/ginkgobioworks/gen/blob/main/src/models/path.rs#L178C12-L178C31

**Comment by dkhofer** [comment]
Overall looks good, but I'd like to take another look once it's been rebased and updated

**Comment by Chris7** [comment]
implemented via #24

**Pull request closed without merging**